packages/preview/minimalbc/0.0.1/README.md

```diff
@@ -0,0 +1,34 @@
+# minimalbc
+
+This repository provides a Typst template for creating sleek and minimalist professional business cards. 
+
+The function, **minimalbc**, allows you to customize the majority of the business card's elements. 
+
+By default, the layout is horizontal. However, it can be easily switched to a vertical layout by passing the value true to the flip argument in the minimalbc function.
+
+Here’s an example of how to use the minimalbc function:
+
+```Typst
+
+#import "lib.typ":minimalbc
```

Comment:
It would be better to use a package import here so that people can copy/paste this example and start from here directly:

```suggestion
#import "@preview/minimalbc:0.1.0": minimalbc
```